frontend/packages/console-app/src/components/quick-starts/controller/QuickStartConclusion.tsx

```diff
@@ -34,8 +35,14 @@ const QuickStartConclusion: React.FC<QuickStartConclusionProps> = ({
           onTaskSelect={onTaskSelect}
         />
       ))}
-      <SyncMarkdownView content={conclusion} />
-      {nextQuickStart && (
+      <SyncMarkdownView
+        content={
+          !hasFailedTask
```

Comment:
@nemesis09 can you please reverse it. It's better to use positivity check, remove the `!`

Reply:
done